rllib/utils/pre_checks/env.py

```diff
@@ -212,6 +213,10 @@ def check_gym_environments(env: Union[gym.Env, "old_gym.Env"]) -> None:
                     space_type,
                 )
             )
+    # sample a valid action in case of parametric actions
+    if isinstance(reset_obs, dict):
```

Comment:
This is a really cool feature of gymnasium, actually :) which I didn't know about.